### ISSUE TITLE
Damaging Ranged Special Attacks Trigger Melee Weapon Skillups

### ISSUE THREAD

**Issue by YourNewfriend** [ccshiro/cc-buglist#1101]
![image](https://cloud.githubusercontent.com/assets/13057166/9148846/277fea22-3d41-11e5-9d01-b0f284e67c16.png)

Scatter Shot raised my Axe Skill, and then shortly after it raised my Bow Skill.  Doesn't affect Concussive Shot apparently.

**Comment by Doppelmoral** [comment]
Your screenshot may show another bug btw:

If there's arrows in your backback and in your quiver, the game will prefer using the backpack ones instead the quiver's. In your pictures it seems your char used backback arrows.

**Comment by YourNewfriend** [comment]
Good observation.

**Comment by ccshiro** [comment]
I've leveled 30 skills in bow now using only scatter shot, and no axe skills yet. Is it possible the mob got close enough to you that your axe swing went off and you scattered at the same time -- cancelling the swing animation -- and just didn't notice the melee swing?

If not possible, this needs reproduction steps.

**Comment by Astixium** [comment]
I tried to reproduce the said Scatter Shot bug with no success. Over 100 Satter Shots where used, none of them gave me a single skill up on any of the seven weapons that I used. From Polearm to Dagger.

On the other hand I found two clipping bugs on Ashkandi, Greatsword of the Brotherhood & Draconic Avenger.

Also the arguement about which arrows does the game use first, the backpacks or the quivers, it's the quiver on Corecraft.

https://youtu.be/OecVdzPQA6E
